cli/flags.rs

```diff
@@ -2093,6 +2093,17 @@ static CONFIG_HELP: Lazy<String> = Lazy::new(|| {
   )
 });
 
+fn task_args<'a>() -> Arg<'a> {
+  Arg::new("config")
+    .short('c')
+    .long("config")
+    .value_name("FILE")
+    .help("Specify the configuration file")
+    .long_help(CONFIG_HELP.as_str())
+    .takes_value(true)
+    .value_hint(ValueHint::FilePath)
```

Comment:
Consider splitting `config_args()` into `config_arg` and `no_config_arg` and applying them to all relevant subcommands. I think this is better than duplicating arg definition